docker_mcp/services/stack/validation.py

```
@@ -624,102 +688,122 @@ async def check_name_conflicts(
+                            "type": "container",
+                            "has_conflict": True,
+                            "error": str(e),
+                        }
+                        conflicting_names.append(f"container:{service_name}")
+
+                # Check network name conflicts
+                for network_name in network_names:
+                    try:
+                        check_cmd = ssh_cmd + [
+                            f"docker network ls --filter name=^{shlex.quote(network_name)}$ --format '{{{{.Name}}}}' | grep -x {shlex.quote(network_name)} && echo 'CONFLICT' || echo 'AVAILABLE'"
+                        ]
```

Ruff (0.14.4)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)